The license model (#1240)
umbraco/UmbracoDocs: Add-ons/The-Licensing-model/index.md

- The development domains works with or without the `www` subdomain
- The license also includes `localhost` as a valid domain

For the Umbraco Courier license you do not need to purchase additional domains if you are running multiple sites within the same Umbraco installation. 

## Configuring and installing your license

You can purchase licenses for our products on our website:

- [Umbraco Forms licenses](https://umbraco.com/products/umbraco-forms/)
- [Umbraco Courier licenses](https://umbraco.com/products/umbraco-courier/)

When you've bought a license you need to configure it with your domains.

You can either configure your license right away, or you can do it later by vising your account on Umbraco.com.

![Configuring Umbraco Forms license](images/configure-forms-license.gif)

## Installing your license

Once you've configured your license with the correct domains, you are ready to install the license on your Umbraco installation.

1. Download your license from your Umbraco.com account - this will give you a `.lic` file
2. Place the file in the `/bin` directory in your Umbraco installation

> The `.lic` file must be placed in the `/bin` directory in order to be registered by Umbraco Courier or Umbraco Forms. Is the file not placed correctly, the application will automatically switch to trial mode.

Reviewer: ```suggestion
The `.lic` file must be placed in the `/bin` directory in order to be registered by Umbraco Courier or Umbraco Forms. If the file isn't placed correctly, the application will automatically switch to trial mode.
```
